--- lesscpy-0.12.0/test/core.py.orig	2016-10-04 11:49:17.000000000 +0200
+++ lesscpy-0.12.0/test/core.py	2016-10-15 10:19:40.728402647 +0200
@@ -24,7 +24,7 @@
     else:
         LESS = glob.glob(os.path.join(_less_path, '*.less'))
     for less in LESS:
-        lessf = less.split('.')[0].split('/')[-1]
+        lessf = less.split('/')[-1].split('.')[0]
         css = os.path.join(_css_path, lessf + '.css')
         if css_minimized:
             mincss = os.path.join(_css_path, lessf + '.min.css')
